diff --git a/src/main/java/com/AI/html/Database.java b/src/main/java/com/AI/html/Database.java
index c98ade9..9992758 100644
--- a/src/main/java/com/AI/html/Database.java
+++ b/src/main/java/com/AI/html/Database.java
@@ -1,14 +1,20 @@
 package com.AI.html;
 
 import java.sql.Connection;
+import java.sql.Date;
 import java.sql.DriverManager;
 import java.sql.PreparedStatement;
 import java.sql.ResultSet;
 import java.sql.SQLException;
 import java.sql.SQLIntegrityConstraintViolationException;
+import java.sql.Statement;
 import java.util.ArrayList;
 import java.util.List;
 
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.ZoneId;
+
 public class Database {
     static final String URL = "jdbc:mysql://localhost:3306/demo";
     static final String USER = "root";
@@ -29,35 +35,36 @@ public class Database {
         }
     }
 
-    public static boolean create(Connection conn, String table, String columns, List<String> columnList) {
+    public static int create(Connection conn, String table, String columns, List<String> columnList) {
         if (conn == null || table == null || table.isEmpty()) {
             System.out.println("Connection is null or table name is invalid.");
-            return false;
+            return 0;
         }
         try (Connection connection = conn) {
             String placeHolders = "";
             for (int i = 1; i <= columnList.size(); i++) {
-                if (i == columnList.size()) {
-                    placeHolders += "?";
-                    break;
-                }
+                // if (i == columnList.size()) {
+                //     placeHolders += "?";
+                //     break;
+                // }
                 placeHolders += "?,";
             }
             PreparedStatement ps = conn
-                    .prepareStatement("INSERT INTO " + table + " (" + columns + ") VALUES (" + placeHolders + ")");
+                    .prepareStatement("INSERT INTO " + table + " (" + columns + ", createDate) VALUES (" + placeHolders + "?)");
             for (int i = 0; i < columnList.size(); i++) {
                 ps.setString(i + 1, columnList.get(i));
             }
+            ps.setString(columnList.size() + 1, getIndianDate());
             int rowsAffected = ps.executeUpdate();
             System.out.println(rowsAffected + " record(s) added.");
 
-            return true;
+            return 1;
         } catch (SQLIntegrityConstraintViolationException e) {
             System.out.println("Record already exists: " + e.getMessage());
-            return false;
+            return -1;
         } catch (Exception e) {
             e.printStackTrace();
-            return false;
+            return -1;
         }
     }
 
@@ -81,12 +88,11 @@ public class Database {
         return true;
     }
 
-    public static boolean delete(Connection conn, int id) {
+    public static boolean delete(Connection conn, String ids) {
         try (Connection connection = conn) {
-            PreparedStatement ps = conn.prepareStatement("Delete from jobs where id=?");
-            ps.setInt(1, id);
-
-            ps.executeUpdate();
+            String sql = "DELETE FROM jobs WHERE id IN (" + ids + ")"; 
+            Statement stmt = conn.createStatement();
+            stmt.executeUpdate(sql);
             System.out.println("Delete");
             return true;
         } catch (Exception e) {
@@ -131,30 +137,38 @@ public class Database {
          */
         // Database.create(conn, table, "role,exp", columnList);5
 
-        conn = Database.connectDB();
+        // conn = Database.connectDB();
         // Database.update(conn, 22, "Super Senior developer", "190 years");
 
-        conn = Database.connectDB();
+        // conn = Database.connectDB();
         /*
          * Retrieve all records from the table
          */
-        // ResultSet list = Database.getAll(conn, table);
+        ResultSet list = Database.getAll(conn, table);
 
         /*
          * Retrieve records from the table based on condition
          */
         // String condition = "id='2'";
-        String condition = "id IN (4,5)";
-        ResultSet list = Database.get(conn, table, condition);
-        try {
-            while (list.next()) {
-                System.out.println("ID: " + list.getInt("id") + ", Role: " + list.getString("role") + ", Exp: "
-                        + list.getString("exp"));
-            }
-        } catch (Exception e) {
-            e.printStackTrace();
-        }
+        // String condition = "id IN (4,5)";
+        // ResultSet list = Database.get(conn, table, condition);
+        // try {
+        //     while (list.next()) {
+        //         System.out.println("ID: " + list.getInt("id") + ", Role: " + list.getString("role") + ", Exp: "
+        //                 + list.getString("exp"));
+        //     }
+        // } catch (Exception e) {
+        //     e.printStackTrace();
+        // }
+
+        // Database.delete(conn, "3,4,5,6,7,8,9,21,22,25,26,27,28.29");
+    }
+
+    public static String getIndianDate() {
+        ZonedDateTime istTime = ZonedDateTime.now(ZoneId.of("Asia/Kolkata"));
 
-        // Database.delete(conn, 24);
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd-MM-yyyy HH:mm:ss a");
+        String formattedTime = istTime.format(formatter);
+        return formattedTime;
     }
 }
diff --git a/src/main/java/com/AI/html/JsBackendAction.java b/src/main/java/com/AI/html/JsBackendAction.java
index fb2e7b7..485b3be 100644
--- a/src/main/java/com/AI/html/JsBackendAction.java
+++ b/src/main/java/com/AI/html/JsBackendAction.java
@@ -1,5 +1,7 @@
 package com.AI.html;
 
+import java.sql.ResultSet;
+
 import javafx.scene.web.WebEngine;
 import javafx.stage.Stage;
 
@@ -13,25 +15,100 @@ public class JsBackendAction {
         this.engine = webEngine;
     }
 
-    public void sayHello() {
-        System.out.println("Button clicked in HTML! Logic running in Java.");
-    }
+    // public void sayHello() {
+    //     System.out.println("Button clicked in HTML! Logic running in Java.");
+    // }
 
-    public void exitApp() {
-        System.out.println("Exiting...");
-        System.exit(0);
-    }
+    // public void exitApp() {
+    //     System.out.println("Exiting...");
+    //     System.exit(0);
+    // }
 
-    public void triggerAction(String actionName) {
-        System.out.println("Java triggered action: " + actionName);
-    }
+    // public void triggerAction(String actionName) {
+    //     System.out.println("Java triggered action: " + actionName);
+    // }
 
-    public void loadAnalytics() {
-        System.out.println("Loading analytics module...");
-    }
+    // public void loadAnalytics() {
+    //     System.out.println("Loading analytics module...");
+    // }
 
     public void navigateTo(String template) {
         Demo demo = new Demo();
         demo.navigateTo(template, this.webStage, this.engine);
     }
+
+    public int createJobs(String role, String exp) {
+        try {
+            var conn = Database.connectDB();
+            var columns = "role,exp";
+            var columnList = new java.util.ArrayList<String>();
+            columnList.add(role);
+            columnList.add(exp);
+            int status = Database.create(conn, "jobs", columns, columnList);
+            if (status > 0) {
+                System.out.println("Job created: " + role + ", " + exp);
+                return status; // Success
+            } else {
+                System.out.println("Job Failed: " + role + ", " + exp);
+                return status; // Failure
+            }
+        } catch (Exception e) {
+            e.printStackTrace();
+            return -1; // Error
+        }
+    }
+
+    public String listJobs() {
+        try {
+            var conn = Database.connectDB();
+            ResultSet list = Database.getAll(conn, "jobs");
+            StringBuilder sb = new StringBuilder();
+            String role, exp, id, created;
+            int index = 1;
+            while (list.next()) {
+                role = list.getString("role");
+                exp = list.getString("exp");
+                created = list.getString("createDate");
+                // id = role +"-"+ exp;
+                id = list.getString("id");
+                sb.append("<tr>\n" + //
+                        "    <td><strong>"+(index++)+"</strong></td>\n" + //
+                        "    <td><strong>"+id+"</strong></td>\n" + //
+                        "    <td>"+role+"</td>\n" + //
+                        "    <td>"+exp+"</td>\n" + //
+                        "    <td>"+created+"</td>\n" + //
+                        "    <td class=\"text-center\">\n" + //
+                        "        <button class=\"btn btn-outline-info btn-sm edit-btn me-1 nav-btn\" data-id=\""+id+"\" data-href=\"edit-job\"><i\n" + //
+                        "                class=\"bi bi-pencil\"></i></button>\n" + //
+                        "        <button class=\"btn btn-outline-danger delete-btn btn-sm nav-btn\" data-id=\""+id+"\"><i class=\"bi bi-trash\"></i></button>\n" + //
+                        "    </td>\n" + //
+                        "</tr>");
+                // sb.append("Role: ").append(list.getString("role")).append(", Experience: ").append(list.getString("exp")).append("\n");
+            }
+
+            if (sb.length() == 0) {
+                return "<tr><td colspan='6' class='text-center'>No jobs found.</td></tr>";
+            }
+            return sb.toString();
+        } catch (Exception e) {
+            e.printStackTrace();
+            return "Error retrieving jobs.";
+        }
+    }
+
+    public Boolean deleteJobs(String id) {
+        try {
+            var conn = Database.connectDB();
+            boolean success = Database.delete(conn, id);
+            if (success) {
+                System.out.println("Jobs deleted: " + id);
+            } else {
+                System.out.println("Job Deletion Failed: " + id);
+            }
+            return success;
+        } catch (Exception e) {
+            e.printStackTrace();
+            return false;
+        }
+    }
 }
diff --git a/src/main/resources/UI/app.css b/src/main/resources/UI/app.css
index 2a254e7..76fb289 100644
--- a/src/main/resources/UI/app.css
+++ b/src/main/resources/UI/app.css
@@ -41,6 +41,16 @@ body {
     min-height: 100vh;
 }
 
+.status {
+    font-weight: 700;
+    display: none;
+}
+
+.status.text-danger,
+.status.text-success {
+    display: block;
+}
+
 /* END: Common CSS*/
 
 
diff --git a/src/main/resources/UI/app.js b/src/main/resources/UI/app.js
index 9defb86..d3d594b 100644
--- a/src/main/resources/UI/app.js
+++ b/src/main/resources/UI/app.js
@@ -6,9 +6,65 @@ $(function () {
     // setActiveNavButton(javaBackend.currentPage);
 
     // Event handlers
-    $('.nav-btn').on('click', navigateTo);
+    $('body').on('click', '.nav-btn', navigateTo);
+    $('.create-btn').on('click', processCreateJob);
+    $('body').on('click', '.delete-btn', processDeleteJob);
+
+    if (!javaBackend['createJobs']) {
+        $('body').append("Status: Backend createJobs function not found.");
+    }
 });
 
 function navigateTo(evt) {
     javaBackend.navigateTo && javaBackend.navigateTo(evt.currentTarget.dataset.href);
+}
+
+function processCreateJob(evt) {
+    // var action = evt.currentTarget.dataset.action;
+    var role = $('#role').removeClass('is-invalid').val();
+    var exp = $('#experience').removeClass('is-invalid').val();
+    $('.status').html('');
+    $('body').append("Status: 1 Processing...");
+
+    if (!role) {
+        $('.status').append('Please select role').addClass('text-danger');
+        $('#role').addClass('is-invalid').focus();
+        return;
+    } else if (!exp) {
+        $('.status').append('Please select experience').addClass('text-danger');
+        $('#experience').addClass('is-invalid').focus();
+        return;
+    }
+
+    $('body').append("Status: 2 Processing...");
+
+    if (javaBackend['createJobs']) {
+        $('body').append("Status: 3 Processing...");
+        var status = javaBackend.createJobs(role, exp);
+        $('body').append("Status: " + status);
+
+        if (status) {
+            $('.status').addClass('text-danger').append('The create job has successful');
+        } else {
+            $('.status').addClass('text-danger').append('The create job has failed');
+        }
+    }
+}
+
+function processDeleteJob(evt) {
+    var $deleteBtn = $(evt.currentTarget);
+    var id = evt.currentTarget.dataset.id;
+    $('.status').html('');
+    // $('body').append("Status: Processing...");
+
+    if (javaBackend['deleteJobs']) {
+        var status = javaBackend.deleteJobs(""+id);
+        $('body').append("Status: " + status);
+        if (status) {
+            // $deleteBtn.closest('tr').remove();
+            location.reload();
+        } else {
+            $('.status').addClass('text-danger').append('The delete job has failed');
+        }
+    }
 }
\ No newline at end of file
diff --git a/src/main/resources/UI/create-job.html b/src/main/resources/UI/create-job.html
index 819dbb6..9c15c99 100644
--- a/src/main/resources/UI/create-job.html
+++ b/src/main/resources/UI/create-job.html
@@ -30,11 +30,6 @@
         </div>
         <div class="card-body p-4">
             <form id="jobForm">
-                <div class="mb-3">
-                    <label for="jobLabel" class="form-label fw-bold">Job Label</label>
-                    <input type="text" class="form-control" id="jobLabel" placeholder="e.g. Senior Backend Engineer" required>
-                </div>
-
                 <div class="mb-3">
                     <label for="role" class="form-label fw-bold">Role</label>
                     <select class="form-select" id="role" required>
@@ -57,32 +52,16 @@
                     </select>
                 </div>
 
+                <div class="mb-4">
+                    <span class="status"></span>
+                </div>
+
                 <div class="mb-4 d-grid1 text-center">
-                    <button type="button" class="btn btn-primary btn-lg create-btn" onclick="processAndRedirect()">
+                    <button type="button" class="btn btn-primary btn-lg create-btn">
                         <i class="bi bi-check-lg me-2"></i>Create Job
                     </button>
-                    <script>
-                        function processAndRedirect() {
-                            // 1. Get text
-                            const textData = document.getElementById("jobLabel");
-
-                            // 2. Send to Backend (Example)
-                            /* fetch('/api/save-data', {
-                                method: 'POST',
-                                headers: { 'Content-Type': 'application/json' },
-                                body: JSON.stringify({ text: textData })
-                            }); */
-
-                            // 3. Redirect to next page
-
-                            window.location.href = "list-job.html";
-                        }
-
-
-
-                    </script>
 
-                    <button type="button" class="btn btn-primary btn-lg nav-btn create-btn" data-href="list-job">
+                    <button type="button" class="btn btn-primary btn-lg nav-btn" data-href="list-job">
                         <i class="bi bi-arrow-left me-2"></i>Back
                     </button>
                 </div>
diff --git a/src/main/resources/UI/edit-job.html b/src/main/resources/UI/edit-job.html
index 3fd8f61..70263d7 100644
--- a/src/main/resources/UI/edit-job.html
+++ b/src/main/resources/UI/edit-job.html
@@ -14,14 +14,16 @@
     <nav class="navbar navbar-expand-lg navbar-white bg-white sticky-top px-20">
         <ul class="nav gap-2">
             <li class="nav-item">
-                <button class="nav-link nav-btn rounded" href="#" data-href="dashboard"><i class="bi bi-house-door me-2"></i> Dashboard</button>
+                <button class="nav-link nav-btn rounded" href="#" data-href="dashboard"><i
+                        class="bi bi-house-door me-2"></i> Dashboard</button>
             </li>
             <li class="nav-item">
                 <button class="nav-link nav-btn active rounded" href="#" data-href="list-job"><i
                         class="bi bi-graph-up me-2"></i> Jobs list</button>
             </li>
             <li class="nav-item">
-                <button class="nav-link nav-btn rounded" href="#" data-href="resume-analyzer"><i class="bi bi-layers me-2"></i> Resume Analyzer</button>
+                <button class="nav-link nav-btn rounded" href="#" data-href="resume-analyzer"><i
+                        class="bi bi-layers me-2"></i> Resume Analyzer</button>
             </li>
         </ul>
     </nav>
@@ -32,11 +34,6 @@
         </div>
         <div class="card-body p-4">
             <form id="jobForm">
-                <div class="mb-3">
-                    <label for="jobLabel" class="form-label fw-bold">Job Label</label>
-                    <input type="text" class="form-control" id="jobLabel" placeholder="e.g. Senior Backend Engineer"
-                        readonly required>
-                </div>
 
                 <div class="mb-3">
                     <label for="role" class="form-label fw-bold">Role</label>
@@ -59,24 +56,13 @@
                         <option value="lead">Lead / Architect (10+ years)</option>
                     </select>
                 </div>
-
-                <div class="mb-4">
-                    <label for="status" class="form-label fw-bold">Status</label>
-                    <select class="form-select" id="status" required>
-                        <option value="" selected disabled>Select Status</option>
-                        <option value="1">Active</option>
-                        <option value="0">De-Active</option>
-                    </select>
-                </div>
-                <option value="lead">Lead / Architect (10+ years)</option>
-                </select>
         </div>
 
         <div class="mb-4 d-grid1 text-center">
-            <button type="button" class="btn btn-primary btn-lg" onclick="submitToJava()">
+            <button type="button" class="btn btn-primary update-btn btn-lg">
                 <i class="bi bi-check-lg me-2"></i>Update Job
             </button>
-            <button type="button" class="btn btn-primary btn-lg nav-btn create-btn" data-href="list-job">
+            <button type="button" class="btn btn-primary btn-lg nav-btn" data-href="list-job">
                 <i class="bi bi-arrow-left me-2"></i>Back
             </button>
         </div>
diff --git a/src/main/resources/UI/list-job.html b/src/main/resources/UI/list-job.html
index b699a24..155afff 100644
--- a/src/main/resources/UI/list-job.html
+++ b/src/main/resources/UI/list-job.html
@@ -32,6 +32,9 @@
                     <i class="bi bi-plus-lg me-1"></i> Add New
                 </button>
             </div>
+                <div class="mb-4">
+                    <span class="status"></span>
+                </div>
             <div class="row mb-3">
                 <div class="col-md-6">
                     <div class="input-group">
@@ -45,7 +48,8 @@
                 <table class="table table-hover align-middle" id="jobTable">
                     <thead class="table-light">
                         <tr>
-                            <th>Job Label</th>
+                            <th>Index</th>
+                            <th>ID</th>
                             <th>Role</th>
                             <th>Experience</th>
                             <th>Created Date</th>
@@ -53,7 +57,7 @@
                         </tr>
                     </thead>
                     <tbody id="jobTableBody">
-                        <tr>
+                        <!-- <tr>
                             <td><strong>Senior Java Developer</strong></td>
                             <td>Software Developer</td>
                             <td>6-10 years</td>
@@ -62,27 +66,7 @@
                                 <button class="btn btn-outline-info btn-sm me-1 nav-btn" data-href="edit-job"><i class="bi bi-pencil"></i></button>
                                 <button class="btn btn-outline-danger btn-sm nav-btn" data-href="edit-job"><i class="bi bi-trash"></i></button>
                             </td>
-                        </tr>
-                        <tr>
-                            <td><strong>Senior Java Developer</strong></td>
-                            <td>Software Developer</td>
-                            <td>6-10 years</td>
-                            <td>Jan 14, 2026</td>
-                            <td class="text-center">
-                                <button class="btn btn-outline-info btn-sm me-1 nav-btn" data-href="edit-job"><i class="bi bi-pencil"></i></button>
-                                <button class="btn btn-outline-danger btn-sm nav-btn" data-href="edit-job"><i class="bi bi-trash"></i></button>
-                            </td>
-                        </tr>
-                        <tr>
-                            <td><strong>Senior Java Developer</strong></td>
-                            <td>Software Developer</td>
-                            <td>6-10 years</td>
-                            <td>Jan 14, 2026</td>
-                            <td class="text-center">
-                                <button class="btn btn-outline-info btn-sm me-1 nav-btn" data-href="edit-job"><i class="bi bi-pencil"></i></button>
-                                <button class="btn btn-outline-danger btn-sm nav-btn" data-href="edit-job"><i class="bi bi-trash"></i></button>
-                            </td>
-                        </tr>
+                        </tr> -->
                     </tbody>
                 </table>
             </div>
@@ -124,6 +108,12 @@
                 `;
             });
         }
+
+        $(function () {
+            if (javaBackend['listJobs']) {
+                $('#jobTableBody').html(javaBackend.listJobs());
+            }
+        });
     </script>
 </body>
 </html>
\ No newline at end of file
